tests/acceptance/addon-dummy-generate-test.js

```diff
@@ -240,4 +240,69 @@ describe('Acceptance: ember generate in-addon-dummy', function () {
     await generateInAddon(['server', '--dummy']);
     expect(file('server/index.js')).to.exist;
   });
+
+  // [WIP] ember addon --lang flag
```

Comment:
Should probably kill this `WIP` comment

```suggestion
  // ember addon --lang flag
```